### ISSUE TITLE
6.5.0 on Hopper

### ISSUE THREAD

**Issue by PhilMiller** [UIUC-PPL/charm#97]
*Original issue: https://charm.cs.illinois.edu/redmine/issues/97*

---
No body.

**Comment by PhilMiller** [comment]
*Original date: 2013-03-28 03:24:22*

---
The known issues blocking some of these builds (#119, #131) have been dealt with. We want to get everything from the 6.5 release wrapped up so we can announce the installed paths for users. If I don't see an issue complete and a path recorded tomorrow, I'm going to start nagging.

**Comment by PhilMiller** [comment]
*Original date: 2013-04-01 17:07:46*

---
Ping: this was supposed to get done last week. Please do the builds and post the path to them so we can close this out.

**Comment by mprobson** [comment]
*Original date: 2013-04-05 00:39:31*

---
All versions (debug, production, trace) of the three compilers (pgi, icc, gcc) built in both smp and non-smp have sucessfuly build mega-test (although smp-pgi-debug required a re-build for some reason). Unsure what I should mark this as due to build errors in Pathscale & Cray versions which were expected. Create a new issue or not worry about it at this time? The builds are all located at /global/u2/m/mprobson/charm/gemini_gni-crayxe-[smp-]<COMPILER>-<TYPE> where COMPILER is one of the three compilers mentioned above and ditto for TYPE.

**Comment by PhilMiller** [comment]
*Original date: 2013-04-05 07:25:00*

---
What's the Cray compiler issue in question? I believe I've successfully built Charm++ with the Cray compiler in the recent past. There's #134 affecting SMP builds and #123 affecting binaries that don't link in the full Charm++ libraries. Do you have something else in mind?

**Comment by mprobson** [comment]
*Original date: 2013-04-08 00:17:32*

---
It appears that everything but the Cray debug build (which is going to be resolved under #167), the Cray SMP builds (#134) and the Pathscale builds works. So, should I mark this as closed or do we need to wait on resolution from those issues?

**Comment by mprobson** [comment]
*Original date: 2013-04-08 06:24:30*

---
I re-built charm/LIBS on Hopper with -g/debug and it worked. It built megatest no problem. I even made a fresh build and tested it and it worked. Seems that they have changed to cce/8.1.3. Should I rebuild all my other cray versions of charm?

**Comment by PhilMiller** [comment]
*Original date: 2013-04-08 16:09:32*

---
Just the cce/debug targets, and then let's be done with this.

**Comment by PhilMiller** [comment]
*Original date: 2013-04-08 21:54:48*

---
What's the path for me to check access to and eventually announce?

**Comment by PhilMiller** [comment]
*Original date: 2013-04-08 22:29:36*

---
Please add builds that support `hugepages` and `persistent` in addition to what's there now. Those will require that you have the module `craype-hugepages8M` loaded before compiling.
